Export/Reports/RevenueExport.php

```diff
@@ -198,7 +198,9 @@ private function getProcesses(): Traversable
             ->andWhere('p.originallyCompletedAt IS NOT NULL');
 
         $startDate = $this->parameters['startdate'] ?? new \DateTime('2001-01-01');
+        $startDate->setTime(00, 00, 00);
         $endDate = $this->parameters['enddate'] ?? new \DateTime('2100-01-01');
+        $endDate->setTime(23, 59, 59);
```

Comment:
This can potentially and theoretically skip a second (or maybe more, cf. https://en.wikipedia.org/wiki/Leap_second).

We should rewrite this to 

```php
$endDate->setTime(00, 00, 00);
$endDate->add(new \DateInterval('P1D'));
```

and update to query accordingly:

```php
  …
  -> andWhere('p.originallyCompletedAt >= :startdate AND p.originallyCompletedAt < :enddate')
  …
```

Furthermore, calling `setTime` on the dates changes the actual date objects which may lead to unexpected side effects when/if they are used outside this function.